extensions-core/hdfs-storage/src/main/java/org/apache/druid/storage/hdfs/tasklog/HdfsTaskLogs.java

```diff
@@ -207,6 +207,37 @@ public void killOlderThan(long timestamp) throws IOException
       }
     }
   }
+
+  /**
+   * Save payload into HDFS, so it can be retrieved later.
+   *
+   * @throws IOException When Druid fails to push the task payload.
+   */
+  @Override
+  public void pushTaskPayload(String taskId, File taskPayloadFile) throws IOException
+  {
+    final Path path = getTaskPayloadFromId(taskId);
+    log.info("Pushing task payload [%s] to location [%s]", taskPayloadFile, path);
```

Comment:
I don't think we need to log the entire payload as an info.

```suggestion
    log.info("Pushing payload for task[%s] to location[%s]", taskId, path);
```